packages/eslint-config-react-native-community/package.json

```diff
@@ -18,7 +18,7 @@
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-flowtype": "^7.0.0",
-    "eslint-plugin-jest": "^25.2.4",
+    "eslint-plugin-jest": "^26.1.3",
```

Comment:
Limt plugin should be separate from jest itself, they are not connected by major version